src/integrations/front-end/feed-improvements.php

```diff
@@ -81,19 +82,48 @@ public function filter_bloginfo_rss( $show, $what ) {
 	}
 
 	/**
-	 * Adds a canonical link header to the main canonical URL for the requested feed object.
+	 * Makes sure send canonical header always runs, because this RRS hook does not support the for_comments parameter
```

Comment:
typo here, I'm fixing
```suggestion
	 * Makes sure send canonical header always runs, because this RSS hook does not support the for_comments parameter
```